src/sagemaker/feature_store/feature_group.py

```diff
@@ -537,6 +538,66 @@ def describe(self, next_token: str = None) -> Dict[str, Any]:
             feature_group_name=self.name, next_token=next_token
         )
 
+    def update(self, feature_additions: Sequence[FeatureDefinition]) -> Dict[str, Any]:
+        """Update a FeatureGroup and add new features from the given feature definitions.
+
+        Args:
+            feature_additions (Sequence[Dict[str, str]): list of feature definitions to be updated.
+
+        Returns:
+            Response dict from service.
+        """
+
+        return self.sagemaker_session.update_feature_group(
+            feature_group_name=self.name,
+            feature_additions=[
+                feature_addition.to_dict() for feature_addition in feature_additions
+            ],
+        )
+
+    def update_feature_metadata(
+        self,
+        feature_name: str,
+        description: str = None,
+        parameter_additions: Sequence[FeatureParameter] = None,
+        parameter_removals: Sequence[str] = None,
+    ) -> Dict[str, Any]:
+        """Update a feature metadata and add/remove metadata.
+
+        Args:
+            feature_name (str): name of the feature to update.
+            description (str): description of the feature to update.
+            parameter_additions (Sequence[Dict[str, str]): list of feature parameter to be added.
+            parameter_removals (Sequence[str]): list of feature parameter key to be removed.
+
+        Returns:
+            Response dict from service.
+        """
+        return self.sagemaker_session.update_feature_metadata(
+            feature_group_name=self.name,
+            feature_name=feature_name,
+            description=description,
+            parameter_additions=[
+                parameter_addition.to_dict() for parameter_addition in parameter_additions
+            ]
+            if parameter_additions is not None
+            else [],
```

Comment:
Do we need this `if/else`?

Reply:
parameter_additions = [parameter_addition.to_dict() for parameter_addition in (parameter_additions or [])]

Unless you have code gaurantines that parameter_additions are not None.

Reply:
@imingtsou parameter_additions can be None and `parameter_addition in parameter_additions` would not work on None type. Error will be `TypeError: 'NoneType' object is not iterable`. Please see unit tests for different use cases. 
@XuetaoYin's code snippet would work but that is analogous to current implementation. Just a different style or personal preference.